gorequest.go

```diff
@@ -1268,8 +1268,7 @@ func (s *SuperAgent) MakeRequest() (*http.Request, error) {
 			req.Host = vals[0]
 		}
 	}
-
-	if len(contentType) != 0 {
+	if len(contentType) != 0 && req.Header.Get("Content-Type") == "" {
```

Comment:
Add comment and point to the bug.

Reply:
I personally don't prefer to clutter the code with references to issues. VCS should provide the necessary history. But, I see that we have used references to github issues elsewhere in the code, so not a big deal I guess to add a line here.

Reply:
You are right. Here, one reason for me is that I feel this should be handled in the following line when GoRequest cannot actually find a correct content-type (and we might need to define a new TargetType).
https://github.com/parnurzeal/gorequest/blob/develop/gorequest.go#L1055

But this would require a lot of change. So, your reference comment would help to get the reason out of this later on.